### PR TITLE
Hw 18

### PR DESCRIPTION
**Pull request by reemaRahhal**
Hi Elias , 
I have done the home work , in my task I built an algorithm that receives an personal details from the user, as required in the lesson (name,age,id,email)  and send them to another functions accordingly. Foe example the algorithm recevies the value of id number which the user has entered  then, the algorithem send the value to function that check if the ID is Valid or Not. 
And I did a documentation for almost each line of code.
At the end I enjoyed doing the task, especially at the stage of building a function of ID integrity.
please review my task

### CODE REVIEW

File: Home_Work_17/js/script.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+ function validEmail() {
```
Comment (EliasSalom) [inline Home_Work_17/js/script.js:1]: this function has been copied from a website please change it and write a new one !!!

**Review by EliasSalom — CHANGES_REQUESTED** [review]
please read my comment and fix the issue!!

File: Home_Work_17/js/script.js (hunk 1 above)
Comment (reemaRahhal) [reply Home_Work_17/js/script.js:1]: I had fixed the function .

**Review by EliasSalom — APPROVED** [review]
Good Work

**Pull request merged**